Source/WebCore/platform/graphics/FontSizeAdjust.h

```diff
@@ -62,8 +62,8 @@ struct FontSizeAdjust {
         IcWidth,
         IcHeight
     } metric;
-    Markable<float, FloatMarkableTraits> value;
     bool isFromFont { false };
+    Markable<float, FloatMarkableTraits> value { };
```

Comment:
Why the `{ }` ? Shouldn't be needed I think and we usually omit it.

Reply:
I agree, I think it's exactly the same (call default ctor) by GCC was complaining about it (cf the previous EWS results).

Reply:
Well I guess the warning is kinda right, there is a "missing initializer" (even if it will still be properly initialised).